bql/lexer/lexer_test.go

```diff
@@ -255,6 +256,28 @@ func TestIndividualTokens(t *testing.T) {
 			[]Token{
 				{Type: ItemLiteral, Text: `"Hallway\"1\""^^type:text`},
 				{Type: ItemEOF}}},
+		{`AFTER 2010-03-10T00:00:00-08:00;`,
+			[]Token{
+				{Type: ItemAfter, Text: "AFTER"},
+				{Type: ItemTime, Text: `2010-03-10T00:00:00-08:00`},
+				{Type: ItemSemicolon, Text: ";"},
+				{Type: ItemEOF}}},
```

Comment:
It is recommended for closing brackets to have the same indentation as the opening brackets, could you change accordingly, please?

Reply:
I agree. Here above I was just following the standard of the file while adding a new test case (as I was suggested in the past to avoid polluting a PR with "cosmetic changes").
By the end, I corrected this indentation above along with others in `lexer_test.go` isolating these improvements in one single commit. Is that ok or should I open another "cosmetic" PR?